libraries/botbuilder-dialogs-adaptive-runtime/src/configurationResourceExplorer.ts

```diff
@@ -2,25 +2,20 @@
 // Licensed under the MIT License.
 
 import { Configuration } from './configuration';
-import { ComponentDeclarativeTypes, FolderResourceProvider, ResourceExplorer } from 'botbuilder-dialogs-declarative';
+import { ComponentDeclarativeTypes, ResourceExplorer } from 'botbuilder-dialogs-declarative';
 import { ok } from 'assert';
 
 export class ConfigurationResourceExporer extends ResourceExplorer {
-    private readonly folderResourceProvider: FolderResourceProvider;
-
     constructor(configuration: Configuration, declarativeTypes: ComponentDeclarativeTypes[]) {
         super({ declarativeTypes });
 
         const applicationRoot = configuration.string(['applicationRoot']);
         ok(applicationRoot);
 
-        this.folderResourceProvider = new FolderResourceProvider(
-            this,
+        this.addFolders(
             applicationRoot,
-            true,
+            ['node_modules'], // Composer copies to `dialogs/imported` so `node_modules` will contain dupes
```

Comment:
I'm surprised `true` didn't throw before. Should we make [this](https://github.com/microsoft/botbuilder-js/blob/9204b41edf06188fc61518a2b90efd62c6e283f5/libraries/botbuilder-dialogs-declarative/src/resources/resourceExplorer.ts#L144) `ignoreFolders.length`, while we're at it?

I'm not sure what the folder structure looks like, but are there other folders that we'd want to ignore?

Reply:
We registered folders in a different way before that did not support an `ignore_folders` param. @srinaath verified that this fix solved the problem so I don't think we need to ignore anything else for now.